async_mock/coroutine.py

```diff
@@ -49,6 +58,11 @@ def unblock(self):
     def mock(self):
         return self.__mock
 
+    async def waitForSingleCall(self):
+        await self.start().wait()
+        self.unblock()
+        await self.end().wait()
+
```

Comment:
Any reason you opt to go through the start() and end() property functions here instead of using self.__start/__end?  I'm also not in love with calling the member variables start and end, since those words are also verbs, so I had to check and see if start() was just a property or if it was performing some side-effect of "starting" something.  Consider renaming these __startingEvent and __endingEvent to make more explicit.